gtwrap/pybind_wrapper.py

```diff
@@ -287,6 +311,19 @@ def wrap_stl_class(self, stl_class):
                     stl_class.properties, cpp_class),
             ))
 
+    def wrap_enum(self, enum, prefix='\n' + ' ' * 8):
+        """Wrap an enum."""
+        namespaces = enum.namespaces()
+        # leverage Typename to get the fully-qualified name easily.
+        name = parser.Typename(namespaces + [enum.name]).to_cpp()
+        res = "    py::enum_<{name}>(m_, \"{enum.name}\", py::arithmetic())".format(
```

Comment:
is this `m_` always appropriate?  (i.e. should it be something else if it's in a different namespace or something?  I'm not sure, the answer might be "no")

I'm guessing this should get `cpp_class` passed in like other functions

Reply:
Yeah good point

Reply:
Done